Adding wizards to command options (#208)
HPInc/ml-git: docs/mlgit_commands.md

```
Usage: ml-git repository storage add [OPTIONS] BUCKET_NAME

  Add a storage BUCKET_NAME to ml-git [This command has a wizard that 

>   will request the necessary information if it is not informed

Reviewer: ```suggestion
  will request the necessary information if it is not informed]
```

Author: The suggestion was applied in 1c1ec60.

>   will request the necessary information if they are not informed

Reviewer 2: ```suggestion
  will request the necessary information if it is not informed
```

Author: The suggestion was applied in 7c084b7.
